ControlFlow/conditional_statements.md

```diff
@@ -0,0 +1,80 @@
+## Conditional Statements
+
+JavaScript conditional statements are used to make decisions in code based on certain conditions. These statements allow your program to execute different blocks of code depending on whether a specified condition evaluates to true or false.
+In JavaScript we have the following conditional statements:
+
+-   Use **if** to specify a block of code to be executed, if a specified condition is true
+-   Use **else** to specify a block of code to be executed, if the same condition is false
+-   Use **else if** to specify a new condition to test, if the first condition is false
+-   Use **switch** to specify many alternative blocks of code to be executed
+
+The most common types of conditional statements in JavaScript are **if**, **else if**, and **else**. Here's an introduction to each:
```

Comment:
unnecesary: The most common types of conditional statements in JavaScript are **if**, **else if**, and **else**. 

Cause it is written one line before: In JavaScript we have the following conditional statements:

-   Use **if** to specify a block of code to be executed, if a specified condition is true
-   Use **else** to specify a block of code to be executed, if the same condition is false
-   Use **else if** to specify a new condition to test, if the first condition is false
-   Use **switch** to specify many alternative blocks of code to be executed